frontend/src/app/components/filters/query-filters/query-filters.component.ts

```diff
@@ -151,8 +147,9 @@ export class QueryFiltersComponent extends UntilDestroyedMixin implements OnInit
     return this.filters[index];
   }
 
-  public isFilterAvailable(id:string):boolean {
-    return (this.wpTableFilters.availableFilters.some(filter => filter.id === id));
+  public isFilterAvailable(filter:QueryFilterResource):boolean {
+    return (this.wpTableFilters.availableFilters.some(availableFilter => availableFilter.id === filter.id) &&
+     !(_.includes(this.wpTableFilters.hidden, filter.id) || filter.isTemplated()));
```

Comment:
No need for lodash here, since it is an array you can use the array.includes native method.